### PR TITLE
Fix double-reporting bug.

### LINKED ISSUE #1: Tests appear to run twice (are reported twice)

### PR DESCRIPTION
**Pull request by insightfuls**
Fixes #1.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -117,8 +117,6 @@ const NWReporter = function(baseReporterDecorator, formatError, injector, emitte
       .map(instantiateReporter.bind(this, injector, locals))
       .filter((reporter) => reporter !== null);
 
-  reporters.forEach((reporter) => emitter.bind(reporter));
```
Comment (kierans) [inline index.js:120]: Good catch.

**Review by kierans — APPROVED** [review]

**Comment by insightfuls** [comment]
@kierans, this code change fixes a double-reporting bug I just encountered. I don't know how you want to handle versioning, so I've just included the code fix here (no version bump or anything).

Please consider making me a collaborator on this and other repositories we are, well, collaborating on, so I can issue pull requests without forking, and on NPM so I can publish modules if/when necessary.

**Comment by kierans** [comment]
@insightfuls This fix has been included in `v1.0.1`. Thanks for the patch.

**Comment by kierans** [comment]
@insightfuls I need a npm username to make you a maintainer for publishing.

**Comment by insightfuls** [comment]
@kierans, it's the same as here: `insightfuls`.

**Pull request merged**